### ISSUE TITLE
Duplicated Group Name

### ISSUE THREAD

**Issue by mostafa901** [EPPlusSoftware/EPPlus#81]
Thanks for this development, it is indeed helpful.

I came up with an excel file which contains groups of pictures, for a reason there are duplicated group Names (only) but the content of each group may differ.
**Group A** { Picture 1, **Picture 2**, Picture 3}
**Group A** {Picture 4, Picture 5, **Picture 2**}

so when you call `sheet.Drawings.Where(o => o is ExcelGroupShape)` it causes an internal error [at this line](https://github.com/EPPlusSoftware/EPPlus/blob/2554f8dfa1c944c0327d5e1271219d6044fd29ac/src/EPPlus/Drawing/ExcelGroupShape.cs#L64)

saying:
```
System.ArgumentException: 'An item with the same key has already been added.'
```
which is logic.

I have temporary fixed it to serve my development, by renaming the duplicated group name before adding to dictionary, then after to save the excel file. 
I tried looking for a way to ungroup through EPPlus, but couldn't find.
I tried also to rename the images inside the groups but I couldn't access as there is already duplicated group name as explained above.

Not sure what i will suggest will be helpful, but I would suggest  adding a function `sheet.GetAllPictures(true)` where true means include pictures inside groups/nested groups.

**Comment by JanKallman** [comment]
Can you provide an Excel file that has this issue?

**Comment by mostafa901** [comment]
sure here download [from here](https://we.tl/t-Dbp5TxpFr8)

**Comment by JanKallman** [comment]
Fixed in version 5.1.1
